--- sh.vim	2010-06-23 02:22:23.557420196 -0400
+++ sh.vim.new	2010-06-23 02:25:44.555305626 -0400
@@ -227,18 +227,9 @@
 syn match   shEscape	contained	'\\.'
 
 " $() and $(()): {{{1
-" $(..) is not supported by sh (Bourne shell).  However, apparently
-" some systems (HP?) have as their /bin/sh a (link to) Korn shell
-" (ie. Posix compliant shell).  /bin/ksh should work for those
-" systems too, however, so the following syntax will flag $(..) as
-" an Error under /bin/sh.  By consensus of vimdev'ers!
-if exists("b:is_kornshell") || exists("b:is_bash")
  syn region shCommandSub matchgroup=shCmdSubRegion start="\$("  skip='\\\\\|\\.' end=")"  contains=@shCommandSubList
  syn region shArithmetic matchgroup=shArithRegion  start="\$((" skip='\\\\\|\\.' end="))" contains=@shArithList
  syn match  shSkipInitWS contained	"^\s\+"
-else
- syn region shCommandSub matchgroup=Error start="\$(" end=")" contains=@shCommandSubList
-endif
 
 if exists("b:is_bash")
  syn cluster shCommandSubList add=bashSpecialVariables,bashStatement
